modules/network-observability-loki-install.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-loki-installation_{context}"]
+
+.Prerequisites
+
+* Installed Loki Operator version 5.6, which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
+* Object storage, such as AWS S3
+* {product-title} 4.10+
+* Linux Kernel 4.18
+
+//* <Any Loki install prerequisites for using with Network Observability operator?>
+
+== Installing the Loki Operator 
+
+You can install the the Loki Operator using the {product-title} web console Operator Hub.
+
+It is recommended to deploy the Loki Operator in the same namespace as the Network Observability operator. You can use the web console or CLI to create a namespace, or new project:
+
+- Using the web console, navigate to the *Project: All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
+- Using the CLI, run the following command to create the`netobserv` namespace: 
++
+[source, terminal]
+----
+$ oc create ns netobserv
+----
+
+.Procedure
+
+. Install the `Loki Operator` Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *Loki Operator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *netobserv*.
+
+.. Verify that you installed the Loki Operator. Visit the *Operators* → *Installed Operators* page and look for *Loki Operator*.
+
+.. Ensure that *Loki Operator* is listed with *Status* as *Succeeded* in all the projects.
++
+. Create a `Secret` YAML file. You can create this secret in the web colsole or CLI. 
+.. Using the web console, navigate to the Imoport icon *+*, in the top right corner. Drop your YAML file into the editor. It is important to create this YAML file in the `netobserv` namespace that uses the `access_key_id` and `access_key_secret` fields to specify your base64-encoded AWS credentials. 
+
+Once you create the secret, you should see it listed under *Workloads* -> *Secrets* in the web console.
++
+The following shows an example secret YAML file:
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: loki-s3
+  namespace: netobserv
+data:
+  access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK <1>
+  access_key_secret: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo= <2>
+  bucketnames: czMtYnVja2V0LW5hbWU= <3>
+  endpoint: aHR0cHM6Ly9zMy5ldS1jZW50cmFsLTEuYW1hem9uYXdzLmNvbQ== <4>
```

Comment:
Are these real secrets? cc @Amoghrd

Reply:
actually the first thing I checked in that PR :rofl:

Reply:
what about using this access key instead: 
`SUxPVkVORVRPQlNFUlYK` 😇